test/test_scytl/test_scytl.py

```diff
@@ -0,0 +1,253 @@
+import unittest
+import os
+import requests_mock
+import csv
+from parsons.scytl import Scytl, scytl
+
+TEST_STATE = 'GA'
+TEST_ELECTION_ID = '114729'
+TEST_VERSION_NUM = '296262'
+
+_dir = os.path.dirname(__file__)
+
+
+class TestScytl(unittest.TestCase):
+
+    @requests_mock.Mocker()
+    def test_get_summary_results_succeeds(self, m):
+        self._mock_responses(m)
+
+        mock_url = scytl.summary_csv_url_template.format(
+            administrator=TEST_STATE, election_id=TEST_ELECTION_ID, version_num=TEST_VERSION_NUM
+        )
+
+        with open(f'{_dir}/114729_summary.zip', 'r') as summary:
+            m.get(mock_url, body=summary)
+
+        scy = Scytl(TEST_STATE, TEST_ELECTION_ID)
+        result = scy.get_summary_results()
+
+        with open(f'{_dir}/114729_summary_expected.csv', 'r') as expected:
```

Comment:
I find it hard to understand if the data is correct since it's hard to compare the source and destination data files in this diff.

Reply:
Yeah that's a reasonable point. I think it's important to use the zip files since unzipping is a big part of what we're testing.

I guess it's somewhat helpful to just read the expected CSV and make sure all the data looks good and reasonable, but not ideal.